notes

```diff
@@ -167,6 +167,33 @@ open_note() {
     $EDITOR "$note_path" < /dev/tty
 }
 
+show_something() {
+    if [[ -p /dev/stdin ]]; then
+        read -d'\n' note_names
+        while read note_name; do
+            show_note "$note_name"
+        done <<< "$note_names"
+    elif [ $# -gt 0 ]; then
+        show_note "$*"
+    else
+        printf "Show requires a name, but none was provided."
+        return 1
+    fi
+}
+
+show_note() {
+    local note_path=$1
+
+    if [[ "$note_path" != *.$NOTES_EXT ]]; then
+        note_path="$note_path.$NOTES_EXT"
+    fi
+    if [ ! -f "$note_path" ]; then
+        note_path="$notes_dir/$note_path"
+    fi
```

Comment:
Both for both of these functions, most of the logic is basically the same as the `open_` equivalent. Can we commonize these, rather the duplicating the `$note_path` and `/dev/stdin` handing?